nodebuilder/util/env.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"os"
+)
+
+var IsBootstrapper bool
+
+func init() {
+	IsBootstrapper = getEnv("BOOTSTRAPPER", "false") == "true"
+}
```

Comment:
[blocking]: this can hit us hard in testground tests. 
Can we do it without `func init()` 🙏

Reply:
Imagine we repeat the same scenario like with sealing config for account prefix balances
This https://github.com/celestiaorg/celestia-node/pull/1583

Caused tg test to fail on sealing issues due to init from both sides (app/node) when used a go.mod for testground tests 

Which caused this PR to appear and later removal of init in node/core/config.go
https://github.com/celestiaorg/celestia-app/pull/1348/

It took a whole month for the proper RC release-fix so the testground test can use RCs without forks 

Proof: https://github.com/celestiaorg/celestia-node/pull/1723

Thus, I propose we stop such practices in node repo that is flooded in sdk/app and rely on different mechanism on how to check/set any Env var in the future.

Reply:
Is it okay to use blank identifiers like [this](https://github.com/celestiaorg/celestia-node/pull/1881/commits/85cb8a533acae96ccc9bd4d7d73c16243c3639d9)? Or do we have the same issue then?

Reply:
These are different stories. The issue you've mentioned can only happen in this case can happen if the key value for env var is the same across different projects. This shouldn't happen between the app and node because the app does not have a notion of a bootstrapper, but a seed node.

Anyway, the node operators may run other software that has such a notion, so we should rename `BOOTSTRAPPER` to `CELESTIA_BOOTSTRAPPER`

Reply:
JFYI, having it as @smuu did it [here](https://github.com/celestiaorg/celestia-node/pull/1881/commits/85cb8a533acae96ccc9bd4d7d73c16243c3639d9) is effectively the same as using init func.

Reply:
I still don't see why removing init changes anything for your case. 
Considering the approve, I am resolving this